velox/common/memory/MemoryPool.cpp

```diff
@@ -703,6 +703,22 @@ int64_t MemoryPoolImpl::usedBytes() const {
   return usedBytes;
 }
 
+int64_t MemoryPoolImpl::releasableReservation() const {
+  std::lock_guard<std::mutex> l(mutex_);
+  if (isLeaf()) {
+    return std::max<int64_t>(
+        0, reservationBytes_ - quantizedSize(usedReservationBytes_));
+  }
+  int64_t releasableBytes{0};
+  for (const auto entry : children_) {
```

Comment:
You need to call visitChildren as children_ is protected by a different lock. Follow the same implementation as usedBytes? thanks!